docs/index.rst

```diff
@@ -71,9 +71,68 @@ Sources
 * Documentation: `<https://move-coop.github.io/parsons/html/index.html>`_
 * Source Code: `<https://github.com/move-coop/parsons>`_
 
+Virtual Environments
+====================
+
+Normally, tools like `pip` install Python libraries directly to your system. Python scripts
+or libraries look for their dependencies in your system. This can cause problems when you
+have two scripts/libraries installed that require differnt versions of the same library.
+
+To solve this problem, we recommend you use *virtual environments* to install Parsons.
+Virtual environments allow you to keep different sets of installed libraries so that you can
+use different versions of the same libraries for different purposes.
+
+Windows
+------------------------------
+
+`Source <https://pypi.org/project/virtualenvwrapper-win/>`_
+
+1. Install virtualenvwrappers from source::
+
+      git clone git://github.com/davidmarble/virtualenvwrapper-win.git
+      cd virtualenvwrapper-win
+      python setup.py install
+
+2. Find the ``Scripts\`` directory for your Python installation, such as ``C:\Users\<User>\AppData\Local\Programs\Python\Python37\Scripts\``.
+
+3. Add the ``Scripts\`` directory to your Path.
+
+4. To create a virtual environment for Parsons, execute this command: ``mkvirtualenv parsons``
+
+5. To use this virtual environment execute: ``workon parsons``
+
+
+Linux / Mac OS
+------------------------------
+
+<Coming Soon>
+
 Installation
 ============
-You can install the most recent release by running: ``pip install parsons``
+
+There are two methods of installing Parsons: Using pip, and from source. Use pip if you just want to install Parsons and start using it. Install from source if you might want to patch Parsons to customize its behavior.
```

Comment:
Another minor edit, but for a slightly smoother sentence, maybe "There are two ways to install Parsons..."